app/frontend/wallet/shelley/shelley-ledger-crypto-provider.ts

```diff
@@ -1,6 +1,6 @@
 import LedgerTransportU2F from '@ledgerhq/hw-transport-u2f'
 import LedgerTransportWebusb from '@ledgerhq/hw-transport-webusb'
-import Ledger, {AddressTypeNibbles} from '@cardano-foundation/ledgerjs-hw-app-cardano'
+import Ledger, * as LedgerTypes from '@cardano-foundation/ledgerjs-hw-app-cardano'
```

Comment:
What do you think about importing the types this way?

Reply:
I personally like it